### ISSUE TITLE
Rating site docs improvements

### ISSUE THREAD

**Issue by wemmyo** [jpmorganchase/salt-ds#6104]
### Related page(s)

https://www.saltdesignsystem.com/salt/components/rating

### Kind of issue

Missing information

### Description

- `Space` should be included in keyboard interaction on accessibility section of site docs
- Live region announcement example on site isn't working as expected
- Custom increments site docs might need to be rewritten to discuss about max number of options, we don't support custom increments which can be confused with precision

### Browser

- [x] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Microsoft Edge

### Operating system

- [x] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Are you a JPMorgan Chase & Co. employee?

- [x] I am an employee of JPMorgan Chase & Co.

**Comment by wemmyo** [comment]
**Screen Reader Issue Feedback:**

When using Mac VoiceOver with the "Clear selection" example:

- Pressing the "Clear" button the first time announces "was cleared." This message could be more descriptive to help users understand what was cleared. Consider updating the announcement to something like "Selection was cleared" or "Your selection has been removed."
- After making a new selection and pressing "Clear" a second time, nothing is announced. The clear action only works the first time.

**Comment by amritadesmet** [comment]
Completed and merged  #6201 requested site doc content updates. Removing myself from this ticket as Mark's covering the Live region update.
